hr_expense_tags/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 ForgeFlow - Elmer García / Joan Solé
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Hr expense tags",
+    "version": "14.0.1.0.0",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/hr-expense",
+    "license": "AGPL-3",
+    "category": "Human Resources",
+    "depends": ["hr_expense", "crm"],
```

Comment:
if "crm" is included because of "crm.tag" usage, then I suggest to use "sales_team" instead.